numpy/core/tests/test_indexerrors.py

```diff
@@ -110,6 +111,16 @@ def assign(obj, ind, val):
         assert_raises(IndexError, lambda: a[(1, [0, 1])])
         assert_raises(IndexError, lambda: assign(a, (1, [0, 1]), 1))
 
+    def test_mapping_error_message(self):
+        a = np.zeros((3, 5))
+        index = (1, 2, 3, 4, 5)
+        try:
```

Comment:
You should use `assert_raises`

Reply:
As far as I know, `assert_raise` can not get the `error message`.